AjaxFetch/js/index.js

```diff
@@ -0,0 +1,34 @@
+values = {
+  users: JSON.parse(localStorage.getItem('users')),
+  posts: JSON.parse(localStorage.getItem('posts')),
+  comments: JSON.parse(localStorage.getItem('comments')),
+  albums: JSON.parse(localStorage.getItem('albums')),
+  photos: JSON.parse(localStorage.getItem('photos'))
+};
+
+/**
+ * Get response from api through 'fetch'
+ * @param {string} path
+ */
+function getResponse(path) {
+  const root = 'https://jsonplaceholder.typicode.com';
+
+  fetch(`${root}/${path}`).then(
+    function (response) {
+      response.json().then(
+        function (data) {
+          localStorage.setItem(path, JSON.stringify(data));
+        });
+    });
+}
```

Comment:
You can separate your callback method into new function and call it like:

```
fetch(`${root}/${path}`)
  .then(response => response.json())
  .then(data => yourCallBackMethod.bind(this, data))
  .catch(error => yourCallBackMethodForFailed.bind(this, error));
```

And decelerate your method line:

```
function yourCallBackMethod(data) {}
```

And one more things, please add catch for all of your Promises. You need to catch errors.
Please read:
https://learn.javascript.ru/promise
https://habrahabr.ru/post/199456/